ETWAnalyzer_uTest/EventDump/DumpTcpTests.cs

```diff
@@ -0,0 +1,112 @@
+                    new Tuple<string, string, Tuple<double, double>>("1000ms", "2000s", new Tuple<double, double>(1, 2000)),
+                };
+
+            foreach (var input in RangeValues)
+            {
+                var args = new string[] { "-dump", "TCP", "-MinMaxConnectionDurationS", input.Item1, input.Item2 };
+                DumpCommand dump = (DumpCommand)CommandFactory.CreateCommand(args);
+                dump.Parse();
+                dump.Run();
+                DumpTcp tcpDumper = (DumpTcp)dump.myCurrentDumper;
+
+                Assert.Equal(input.Item3.Item1, tcpDumper.MinMaxConnectionDurationS.Min);
+                Assert.Equal(input.Item3.Item2, tcpDumper.MinMaxConnectionDurationS.Max);
+
+            }
+
+        }
+
+        public List<string> Messages { get; set; } = new List<string>();
+
+        private ITestOutputHelper myWriter;
+
+        public DumpTcpTests(ITestOutputHelper myWriter)
+        {
+            this.myWriter = myWriter;
+        }
+
+        internal void Add(string message)
+        {
+            Messages.Add(message);
+        }
+
+        static readonly SocketConnection ipRemPort = new("123.205.91.10", 10000);
+        static readonly SocketConnection ipSrcPort = new("123.456.789.10", 100);
+
+        static readonly DateTimeOffset timeStampOpen = new DateTimeOffset(2023, 8, 11, 10, 05, 00, TimeSpan.Zero);
+        static readonly DateTimeOffset timeStampClose = new DateTimeOffset(2023, 8, 11, 10, 05, 00, TimeSpan.Zero);
+        static readonly DateTimeOffset timeStampClose1 = new DateTimeOffset(2023, 8, 11, 10, 05, 10, TimeSpan.Zero);
+        static readonly DateTimeOffset timeStampClose2 = new DateTimeOffset(2023, 8, 11, 10, 05, 20, TimeSpan.Zero);
+        static readonly TcpConnection tcpConnection = new(123, ipRemPort, ipSrcPort, timeStampOpen, timeStampClose, "", 100, 120, 100, 100, ETWProcessIndex.Invalid);
+        static readonly TcpConnection tcpConnection1 = new(456, ipRemPort, ipSrcPort, timeStampOpen, timeStampClose1, "", 100, 120, 100, 100, ETWProcessIndex.Invalid);
+        static readonly TcpConnection tcpConnection2 = new(789, ipRemPort, ipSrcPort, timeStampOpen, timeStampClose2, "", 100, 120, 100, 100, ETWProcessIndex.Invalid);
+        
+        
+
+        List<DumpTcp.MatchData> CreateTestData()
+        {
+            List<DumpTcp.MatchData> data = new List<DumpTcp.MatchData>
+            { 
+                new DumpTcp.MatchData
+                {
+                    Connection = tcpConnection,
+                },
+                new DumpTcp.MatchData
+                {
+                    Connection = tcpConnection1,
+                },
+                new DumpTcp.MatchData
+                {
+                    Connection = tcpConnection2,
+                }
+            };
+
+            return data;
+        }
+
+        [Fact]
+        public void MinMaxConnectionDurationFilterTest()
```

Comment:
Please create two tests. One with matching values and the other with not matching like
ConnectionDuration_Matching()
ConnectionDurationFilter_NotMatching()

to test the corresponding corner cases.